lib/linter-coffee-variables.coffee

```diff
@@ -103,12 +110,14 @@ _addOriginalCodePosition = (sourceMap, variables) -> (error) ->
 
 
 _lookUpVariablePosition = (variableName, variables) ->
-  variables
-    .filter (v) ->
-      v.name is variableName
-    .map (v) ->
-      line   : v.line
-      column : v.column
+  mappedVariables = []
+  for v in variables
```

Comment:
Just run this in node and see for yourself.
http://codereview.stackexchange.com/questions/49549/performance-for-foreach-and-for-key